template/apps/scrutiny.json

```diff
@@ -0,0 +1,36 @@
+{
+	"categories": [
+		"Other",
+		"Tools"
+	],
+	"description": "WebUI for smartd S.M.A.R.T monitoring",
+	"image_arm32": "ghcr.io/analogj/scrutiny:master-omnibus",

```

Comment:
Just remove this line then this wont list on arm32 template

Reply:
> Just remove this line then this (scrutiny) wont list on arm32 template

Done!